python/example_code/s3/s3_basics/presigned_url.py

```diff
@@ -83,9 +86,7 @@ def usage_demo():
             )
 
     if response is not None:
-        print("Got response:")
         print(f"Status: {response.status_code}")
```

Comment:
```suggestion
        print(f"Status: {response.status_code}\nReason: {response.reason}")
```
Just add the reason for better diagnostics.

Reply:
Thanks for your feedback.